src/main/js/EditorPage.jsx

```diff
@@ -119,14 +119,17 @@ class PipelineLoader extends React.Component {
     componentWillMount() {
         pipelineStore.setPipeline(null); // reset any previous loaded pipeline
         const { organization, pipeline } = this.props.params;
-        let href = Paths.rest.pipeline(organization, pipeline);
+        const split = pipeline.split('/');
+        const team = split[0];
+        let href = Paths.rest.pipeline(organization, team);
+
         pipelineService.fetchPipeline(href, { useCache: true })
             .then(pipeline => {
                 if(pipeline.scmSource && pipeline.scmSource.id) {
                     this.setState({ scmId: pipeline.scmSource.id });
                 }
-                this.loadPipeline();
             });
+        this.loadPipeline();
```

Comment:
I don't think this should be moved outside, `loadPipline` needs `scmId` on line 176